src/main/java/au/org/democracydevelopers/raireservice/controller/AssertionController.java

```diff
@@ -99,12 +99,13 @@ public ResponseEntity<GenerateAssertionsResponse> serve(@RequestBody GenerateAss
     logger.debug(String.format("%s Calling raire-java with assertion generation request.",prefix));
     RaireResultOrError solution = generateAssertionsService.generateAssertions(request);
 
```

Comment:
If the persisting is done on line 102, then I think we need to change the comment on 105 as we have already saved them to the database.